pom.xml

```diff
@@ -145,6 +145,7 @@
         <guava.version>31.1-jre</guava.version>
         <guava.failureaccess.version>1.0.1</guava.failureaccess.version>
         <hadoop.version>3.3.4</hadoop.version>
+        <hadoop.client.api.version>3.3.5</hadoop.client.api.version>
```

Comment:
why introduce a new property instead of changing `hadoop.version` directly?

Reply:
if change the hadoop.version, it will change the version of hadoop-client-runtime , hadoop-minikdc,hadoop-client-minicluster etc. And are you sure about this ?

Reply:
yes, we should keep them same version

Reply:
ok, i will sumbit again.